pkgs/applications/editors/atom/default.nix

```diff
@@ -36,7 +36,7 @@ let
         --set-rpath "${atomEnv.libPath}" \
         $share/resources/app/apm/bin/node
       patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        $out/share/atom${lib.optionalString beta "-beta"}/resources/app.asar.unpacked/node_modules/symbols-view/vendor/ctags-linux
+        $out/share/${pname}/resources/app.asar.unpacked/node_modules/symbols-view/vendor/ctags-linux
```

Comment:
You should replace `$out/share/${pname}` with `$share`.

Reply:
Oh, yeah, right you are